File: panipaniRacing/Assets/Game 5/Script/Game5_Number.cs
using UnityEngine;
using System.Collections;

public class Game5_Number : MonoBehaviour
{
	public float speed;			//Move speed
	public float destroyTime;	//Life time
	
	void Start ()
	{
		//Start DestroyGO
		StartCoroutine("DestroyGO");
	}
	
	void Update ()
	{
		//Move up
		transform.Translate(Vector3.up * speed * Time.smoothDeltaTime);
	}
	
	IEnumerator DestroyGO()
	{
		//Wait 
		yield return new WaitForSeconds(destroyTime);
		//Kill
		Destroy(gameObject);
	}
}

File: panipaniRacing/Assets/Game 5/Script/Javascript/Game5_Number.js
	public var speed : float;		//Move speed
	public var destroyTime : float;	//Life time
	
	function Start ()
	{
		//Start DestroyGO
		StartCoroutine("DestroyGO");
	}
	
	function Update ()
	{
		//Move up
		transform.Translate(Vector3.up * speed * Time.smoothDeltaTime);
		
	}
	
	function DestroyGO()
	{
	
		//Wait
		yield WaitForSeconds(destroyTime);
		//Kill
		Destroy(gameObject);
		
	}